### PR TITLE
Package osnap.0.1.1

### PR DESCRIPTION
**Pull request by vch9**
### `osnap.0.1.1`
OCaml random snapshot testing

---
* Homepage: https://github.com/vch9/osnap
* Source repo: git+https://github.com/vch9/osnap.git
* Bug tracker: https://github.com/vch9/osnap/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/osnap/osnap.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+authors: [
+ "Valentin Chaboche <valentin.chaboche@nomadic-labs.com>"
+]
+
+depends: [
+  "dune" {>= "2.8.0" }
+  "ocaml" {>= "4.12.0" }
+  "qcheck" {>= "0.17" }
+  "patdiff" { >= "0.14.0" }
```
Comment (mseri) [inline packages/osnap/osnap.0.1.1/opam:20]:
```suggestion
  "patdiff" { >= "v0.14.0" }
```

Comment (mseri) [reply packages/osnap/osnap.0.1.1/opam]: Please upstream this change

**Comment by kit-ty-kate** [comment]
Thanks! Sorry for the delay

**Comment by vch9** [comment]
Thanks both of you!

File: packages/osnap/osnap.0.1.1/opam (hunk 1 above)
Comment (vch9) [reply packages/osnap/osnap.0.1.1/opam]: Will do, thank you :)

**Pull request merged**